packages/format-library/src/image/index.js

```diff
@@ -145,10 +145,20 @@ function Edit( {
 	activeObjectAttributes,
 	contentRef,
 } ) {
+	let currentImageId;
+	if ( isObjectActive && activeObjectAttributes.className ) {
+		const wpImageMatch =
+			activeObjectAttributes.className.match( /wp-image-(\d+)/ );
+		if ( wpImageMatch && wpImageMatch[ 1 ] ) {
+			currentImageId = parseInt( wpImageMatch[ 1 ], 10 );
+		}
+	}
+
 	return (
 		<MediaUploadCheck>
 			<MediaUpload
 				allowedTypes={ ALLOWED_MEDIA_TYPES }
+				value={ currentImageId }
```

Comment:
```suggestion
				value={ getCurrentImageId( activeObjectAttributes ) }
```

The `activeObjectAttributes` is an empty object, when `isObjectActive` is false, so the method will return `undefined` as expected.